### DISCUSSION TITLE
How to Kill Active Selenium Sessions Exceeding Specified Time?

### DISCUSSION THREAD

**Discussion by johndaniel7** [SeleniumHQ/docker-selenium#2412] [Q&A]
I have encountered a situation where a test hangs indefinitely while waiting for a specific condition to be met (in this case, waiting for a global loading overlay). The session remains active, and the test keeps retrying without any progress.i have enabled the SE_ENABLE_BROWSER_LEFTOVERS_CLEANUP=true environment variable but there is no effect. Can you suggest any environment variable in Selenium Docker compose [https://github.com/SeleniumHQ/docker-selenium/blob/trunk/docker-compose-v3-video-upload.yml](url) to automatically kill active Selenium sessions or tests that exceed a certain duration? Thanks

**Comment by VietND96** [comment]
Env var `SE_NODE_SESSION_TIMEOUT` (unit in seconds, default: 300s). What is the value that you set for this env var?
